### PR TITLE
886573-UG content for Smart data label alignment support - Circular chart

### PR DESCRIPTION
**Pull request by EzhilarasanElangovan31**
**Description**

Added UG content for Smart data label alignment support - Circular chart

### CODE REVIEW

File: MAUI/Circular-Charts/DataLabels.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ this.Content = chart;
 
 ## LabelTemplate
 
-The [SfCircularChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCircularChart.html) provides support to customize the appearance of the datalabel using the [LabelTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSeries.html#Syncfusion_Maui_Charts_ChartSeries_LabelTemplate) property.
+The [SfCircularChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCircularChart.html) provides support to customize the appearance of the data label using the [LabelTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSeries.html#Syncfusion_Maui_Charts_ChartSeries_LabelTemplate) property.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Circular-Charts/DataLabels.md:146]: @EzhilarasanElangovan31 is this changes needed?

Comment (EzhilarasanElangovan31) [reply MAUI/Circular-Charts/DataLabels.md:146]: CI failed due to spelling error, so I modified it.

File: MAUI/Circular-Charts/DataLabels.md [hunk 2]
```diff
@@ -222,4 +222,63 @@ this.Content = chart;
...
+
+* [Shift]() - smartly arranges the overlapped data labels.
+* [Hide]() - hides the intersected data labels.
+* [None]() - intersected data labels will be visible.
+
+By default, value for [SmartLabelAlignment]() property [Shift]().
+
+If the [LabelPlacement](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartDataLabelSettings.html?tabs=tabid-5%2Ctabid-3%2Ctabid-1#Syncfusion_Maui_Charts_ChartDataLabelSettings_LabelPlacement) is [Inner](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.DataLabelPlacement.html#Syncfusion_Maui_Charts_DataLabelPlacement_Inner) and the [SmartLabelAlignment]() is [Shift](), then the overlapped labels will shift to outside the slices and arrange smartly. If the [LabelPlacement](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartDataLabelSettings.html#Syncfusion_Maui_Charts_ChartDataLabelSettings_LabelPlacement) is [Inner](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.DataLabelPlacement.html#Syncfusion_Maui_Charts_DataLabelPlacement_Inner) and the [SmartLabelAlignment]() is [Hide](), then the overlapped labels will be hidden.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Circular-Charts/DataLabels.md:237]: @EzhilarasanElangovan31 For circular chart positioning, we have exposed new API - LabelPosition. Modify the content based on it

File: MAUI/Circular-Charts/DataLabels.md [hunk 3]
```diff
@@ -222,4 +222,63 @@ this.Content = chart;
...
+{% highlight xaml %}
+
+<chart:SfCircularChart>
+
+    <chart:PieSeries ItemsSource="{Binding Data}" 
+                     ShowDataLabels="True"
+                     XBindingPath="Product" 
+                     YBindingPath="SalesRate">
+         <chart:PieSeries.DataLabelSettings>
+             <chart:CircularDataLabelSettings LabelPlacement="Outer" SmartLabelAlignment="Shift"/>
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Circular-Charts/DataLabels.md:256]: @EzhilarasanElangovan31 Similarly, change the API name and its Enum value

**Comment by VimalaThirumalaiKumarSF3739** [comment]
@EzhilarasanElangovan31 ConnectorLineStyle content missing, update it

**Comment by VimalaThirumalaiKumarSF3739** [comment]
@EzhilarasanElangovan31 update images also

File: MAUI/Circular-Charts/DataLabels.md (hunk 2 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Circular-Charts/DataLabels.md]: Modified

File: MAUI/Circular-Charts/DataLabels.md (hunk 3 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Circular-Charts/DataLabels.md]: changed the API name and its Enum value

File: MAUI/Circular-Charts/DataLabels.md [hunk 4]
```diff
@@ -141,9 +141,87 @@ this.Content = chart;
 
 ![DataLabel context in MAUI Chart](DataLabel_images/maui_chart_datalabel_context.png)
 
+## Connector line style
+
+This feature is used to customize the connector line. The following [ConnectorLineStyle]() property are used to customize the connector line.
+
+* [Stroke]() – used to change the color of the line
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Circular-Charts/DataLabels.md:148]: @EzhilarasanElangovan31 modify the property as - Gets or sets the border stroke color of the connector line.

File: MAUI/Circular-Charts/DataLabels.md [hunk 5]
```diff
@@ -222,4 +300,76 @@ this.Content = chart;
 
 {% endtabs %}
 
-![Label template in MAUI Chart](DataLabel_images/maui_chart_datalabel_template.png)
\ No newline at end of file
+![Label template in MAUI Chart](DataLabel_images/maui_chart_datalabel_template.png)
+
+## Smart Labels
+
+This feature is used to arrange the data labels smartly and avoid the intersection when there is overlapping of labels. The [SmartLabelAlignment]() enum contains the following values.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Circular-Charts/DataLabels.md:307]: @EzhilarasanElangovan31  In MAUI, we don't mention as feature in the content. We will be using the property and explain its usage. Check MAUI UG and modify.

File: MAUI/Circular-Charts/DataLabels.md [hunk 6]
```diff
@@ -222,4 +300,76 @@ this.Content = chart;
...
+
+By default, value for [SmartLabelAlignment]() property [Shift]().
+
+If the [LabelPosition]() is [Inside]() and the [SmartLabelAlignment]() is [Shift](), then the overlapped labels will shift to outside the slices and arrange smartly. If the [LabelPosition]() is [Inside]() and the [SmartLabelAlignment]() is [Hide](), then the overlapped labels will be hidden.
+
+If the [LabelPosition]() is [Outside]() and the [SmartLabelAlignment]() is [Shift](), then the overlapped labels arrange smartly. If the [LabelPosition]() is [Outside]() and the [SmartLabelAlignment]() is [Hide](), then the overlapped labels will be hidden.
+
+If the [SmartLabelAlignment]() is [None](), then the overlapped labels will be visible irrespective of LabelPosition.
+
+When the [SmartLabelAlignment]() is [Shift](), and if the data label goes out of the chart area, then the labels got trimmed and the tooltip is shown when clicking/tapping the data label.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Circular-Charts/DataLabels.md:321]: @EzhilarasanElangovan31 why need to mention about tooltip?

File: MAUI/Circular-Charts/DataLabels.md [hunk 7]
```diff
@@ -222,4 +300,76 @@ this.Content = chart;
...
+};
+
+chart.Series.Add(series);
+this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+![Smart Labels in MAUI Chart](DataLabel_images/maui_chart_smart_datalabel.png)
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Circular-Charts/DataLabels.md:375]: @EzhilarasanElangovan31 this image is missing

File: MAUI/Circular-Charts/DataLabels.md [hunk 8]
```diff
@@ -141,9 +141,87 @@ this.Content = chart;
 
 ![DataLabel context in MAUI Chart](DataLabel_images/maui_chart_datalabel_context.png)
 
+## Connector line style
+
+This feature is used to customize the connector line. The following [ConnectorLineStyle]() property are used to customize the connector line.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Circular-Charts/DataLabels.md:146]: @EzhilarasanElangovan31  In MAUI, we don't mention as feature in the content. We will be using the property and explain its usage. Check MAUI UG and modify.

File: MAUI/Circular-Charts/DataLabels.md (hunk 4 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Circular-Charts/DataLabels.md]: Modified

File: MAUI/Circular-Charts/DataLabels.md (hunk 5 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Circular-Charts/DataLabels.md]: Modified the content

File: MAUI/Circular-Charts/DataLabels.md (hunk 7 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Circular-Charts/DataLabels.md:375]: Modified the already added image

**Comment by VimalaThirumalaiKumarSF3739** [comment]
Closing the PR as duplicate, original PR https://github.com/syncfusion-content/maui-docs/pull/2345
